File: customers-staff-master/src/CustomersAndStaff/FrontApi.Dto/WorkOrders/VehicleCustomerValueDto.cs
namespace Market.CustomersAndStaff.FrontApi.Dto.WorkOrders
{
    public class VehicleCustomerValueDto : BaseCustomerValueDto
    {
        public string Vin { get; set; }
        public string Brand { get; set; }
        public string Model { get; set; }
        public string RegisterSign { get; set; }
        public int? Year { get; set; }
        public string BodyNumber { get; set; }
        public string EngineNumber { get; set; }
    }
}

File: customers-staff-master/src/Front/src/api/new/dto/WorkOrder/VehicleCustomerValueDto.ts
import { BaseCustomerValueDto } from "./BaseCustomerValueDto";

export interface VehicleCustomerValueDto extends BaseCustomerValueDto{
  vin: string;
  brand: string;
  model: string;
  registerSign: string;
  year?: number;
  bodyNumber: string;
  engineNumber: string;
}
